client.go

```diff
@@ -379,34 +379,53 @@ func (c *Client) Mail(from string, opts *MailOptions) error {
 	if err := c.hello(); err != nil {
 		return err
 	}
-	cmdStr := "MAIL FROM:<%s>"
+
+	var sb strings.Builder
+	sb.Grow(510 + 14 + 26 + 11 + 9 + 9 + 39 + 500)
```

Comment:
Hm, this seems a bit like micro-optimization with these magic constants... I'd prefer to leave this out or just pick a reasonably high enough power of 2 value.